documentation/getting_started/kubernetes.adoc

```diff
@@ -16,21 +16,20 @@ include::prerequisites-kubernetes.adoc[leveloffset=+1]
 
 . Download one of the releases from https://github.com/EnMasseProject/enmasse/releases and unpack it.
 
-. Create the namespace where you want to deploy {ProductName}
+. (Optional) Replace namespace if you want to deploy to something other than `{ProductNamespace}` (and substitute `{ProductNamespace}` in subsequent steps):
```

Comment:
"If you want to install and deploy {ProductName} to a namespace other than `{ProductNamespace}` run the following command:" 

For the  "(and substitute `{ProductNamespace}` in subsequent steps)" part, for subsequent commands suggest styling `{ProductNamespace}` as a variable and adding the following phrase after the affected commands: "where `<{ProductNamespace}>` is the namespace where {ProductName} is installed."